src/main/java/com/example/springresttask/model/UploadedFile.java

```diff
@@ -0,0 +1,43 @@
+package com.example.springresttask.model;
+
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class UploadedFile {
+
+    int fileId;
+    String fileName;
+    String fileType;
+    byte[] fileData;
+    String path;
+
+    public UploadedFile(int id, MultipartFile file) {
+        fileId = id;
+        fileName = file.getOriginalFilename();
+        int lastIndex = fileName.lastIndexOf('.');
```

Comment:
This class should be a stupid DTO that does nothing. 
You can move the logic of getting a file's extension to a static method in some utility class